### ISSUE TITLE
Lag in Solar Orbit

### ISSUE THREAD

**Issue by PresentMonkey** [Gameslinx/Tessellation#47]
When in solar orbit the game gets really laggy. I have looked in the log and Parallax is the culprit. The lag is gone if I uninstall Parallax. 

```
[EXC 12:37:40.755] NullReferenceException: Object reference not set to an instance of an object
	ParallaxShader.Position.Update () (at <a80f532d3c7549bf8e89adcd6559f787>:0)
	UnityEngine.DebugLogHandler:LogException(Exception, Object)
	ModuleManager.UnityLogHandle.InterceptLogHandler:LogException(Exception, Object)
	UnityEngine.Debug:CallOverridenDebugHandler(Exception, Object)
```
	
This happens every second when I'm in solar orbit

**Comment by PresentMonkey** [comment]
Forgot to add that I'm on Linux.

**Comment by EmeraldWither** [comment]
A new version of Parallax has been released. Try using that, and see if the issue still persists.

**Comment by Gameslinx** [comment]
This is not the source of the lag, but the log spam has been fixed. Will be released in 1.2.3
